jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -502,5 +509,91 @@ public int encodeASCII(char[] sa, int sp, byte[] da, int dp, int len) {
 	}
 /*[ENDIF] JAVA_SPEC_VERSION >= 17 */
 
+/*[IF JAVA_SPEC_VERSION >= 19]*/
+	public Thread currentCarrierThread() {
+		return Thread.currentCarrierThread();
+	}
+
+	public <T> T getCarrierThreadLocal(ThreadLocal<T> local) {
+		return local.getCarrierThreadLocal();
+	}
+
+	public <T> void setCarrierThreadLocal(ThreadLocal<T> local, T value) {
+		local.setCarrierThreadLocal(value);
+	}
+
+	public <V> V executeOnCarrierThread(Callable<V> task) throws Exception {
+		// TODO
+		return null;
```

Comment:
Better to throw for these todos than to return null as it's many times easier to find the throwing code than the unexpected null.

The only exception to this is if this API is used and null is a valid return while we bootstrap